commands/apps/open.js

```diff
@@ -0,0 +1,51 @@
+'use strict';
+
+let co       = require('co');
+let cli      = require('heroku-cli-util');
+let extend   = require('util')._extend;
+let url      = require('url');
+
+function* run (context, heroku) {
+  let app, path;
+  if (context.args.path) {
+    app  = context.args.app;
+    path = context.args.path;
+  } else if (context.args.app && context.app) {
+    app = context.app;
+    path = context.args.app;
+  } else if (context.args.app) {
+    app = context.args.app;
+  } else {
+    app = context.app;
+  }
+  if (!app) throw new Error('No app specified');
+  app = yield heroku.apps(app).info();
+  yield cli.open(url.resolve(app.web_url, path || ''));
+}
+
+let cmd = {
+  topic: 'apps',
+  command: 'open',
+  description: 'open the app in a web browser',
+  help: `
+Examples:
+
+  $ heroku open myapp
```

Comment:
@dickeyxxx my expectation was that this would open the path `myapp`, which would match the behavior of the old command.

https://github.com/heroku/heroku/blob/master/lib/heroku/command/apps.rb#L227